pytorch_pfn_extras/onnx/pfto_exporter/export.py

```diff
@@ -221,6 +229,8 @@ def onnx_node_doc_string(onnx_node: torch._C.Node, torch_node: torch._C.Node) ->
     torch.float16: onnx.TensorProto.DataType.FLOAT16,  # type: ignore[attr-defined]
     torch.complex64: onnx.TensorProto.DataType.COMPLEX64,  # type: ignore[attr-defined]
     torch.complex128: onnx.TensorProto.DataType.COMPLEX128,  # type: ignore[attr-defined]
+    torch.torch.float8_e4m3fn:  onnx.TensorProto.DataType.FLOAT8E4M3FN,  # type: ignore[attr-defined]
+    torch.torch.float8_e5m2:  onnx.TensorProto.DataType.FLOAT8E5M2,  # type: ignore[attr-defined]
```

Comment:
https://github.com/pfnet/pytorch-pfn-extras/actions/runs/6702797789/job/18212320599?pr=781#step:8:379
```suggestion
    torch.float8_e4m3fn:  onnx.TensorProto.DataType.FLOAT8E4M3FN,  # type: ignore[attr-defined]
    torch.float8_e5m2:  onnx.TensorProto.DataType.FLOAT8E5M2,  # type: ignore[attr-defined]
```

Reply:
Thanks. It seems we need to check versions of pytorch and onnx: 98c54754b0e1a5b78c2d237e91874f09f47056c2